### PR TITLE
Add scene status message request store command

### PR DESCRIPTION
**Pull request by maximilianriemensberger**
#36

### CODE REVIEW

File: pypck/pck_commands.py [hunk 1]
```diff
@@ -1011,6 +1018,29 @@ def change_scene_register(register_id: int) -> str:
...
 
+    @staticmethod
+    def store_scene_outputs_direct(
+        register_id: int, scene_id: int, percents: Sequence[float], ramps: Sequence[int]
+    ) -> str:
+        """Store the given output values and ramps in the given scene.
+
+        :param    int           scene_id:   Scene id 0..9
+        :param    list(float)   percents:   Output values in percent as list
+        :param    list(int)     ramp:       Ramp values as list
```
Comment (alengwenus) [inline pypck/pck_commands.py:1029]: Documentation for register_id is missing.

File: pypck/module.py [hunk 2]
```diff
@@ -1018,6 +1018,29 @@ async def store_scene(
...
+        register_id: int,
+        scene_id: int,
+        percents: Sequence[float],
+        ramps: Sequence[int],
+    ) -> bool:
+        """Store the given output values and ramps in the given scene.
+
+        :param    int           scene_id:   Scene id 0..9
+        :param    list(float)   percents:   Output values in percent as list
+        :param    list(int)     ramp:       Ramp values as list
```
Comment (alengwenus) [inline pypck/module.py:1032]: Documentation for register_id is missing.

**Review by alengwenus — CHANGES_REQUESTED** [review]
Thanks @maximilianriemensberger. Just two docstrings need some attention.

**Comment by maximilianriemensberger** [comment]
@alengwenus Fixed. Thanks!

**Review by alengwenus — APPROVED** [review]

**Pull request merged**